index.html

```diff
@@ -0,0 +1,150 @@
+                a new level.</p>
+            </div>
+            <div class="program-tag flex">
+              <i class="program-img fa-solid fa-dumbbell card-icon"></i>
+              <h3 class="program-title">Equipment</h3>
+              <p class="program-detail">We offer a large selection of cardiovascular, strength and free weight
+                equipment. You are sure to find some
+                of your favorites and try out new options to jazz up your workout.</p>
+            </div>
+            <div class="program-tag flex">
+              <i class="program-img fa-solid fa-person-rays card-icon"></i>
+              <h3 class="program-title">Exercise Classes</h3>
+              <p class="program-detail">Enjoy fun and challenging group fitness classes led by certified motivating
+                instructors</p>
+            </div>
+            <div class="program-tag flex">
+              <i class="program-img fa-solid fa-user-nurse card-icon"></i>
+              <h3 class="program-title">Health Assessment & Prescription</h3>
+              <p class="program-detail">Meet one on one with a degreed exercise professional to assess your fitness
+                status and set a personalized
+                exercise plan based on your goals</p>
+            </div>
+          </div>
+          <button type="button" class="join-btn">Join SPFC 2023 Now</button>
+          <a href="#" class="desktop-link max-font-26">see the whole program</a>
+        </div>
+      </div>
+    </section>
+
+    <section id="trainers" class="flex column">
+      <div class="wrapper flex column">
+        <h2 class="title max-font-26">Featured Trainers</h2>
+        <div id="trainers-container" class="trainers-container flex column">
+
+        </div>
+        <button type="button" class="trainer-btn"><span class="moreless">MORE </span><i
+            class="chevron fas fa-chevron-down"></i></button>
+      </div>
+    </section>
+
+    <section id="partners">
+      <div class="container">
+        <h2 class="title max-font-26">Partner</h2>
+        <div class="company flex">
+          <span id="facebook">Facebook</span>
+          <span id="mozilla">Mozilla</span>
+          <span id="apple">Apple</span>
+          <span id="microsoft">Microsoft</span>
+          <span id="google">Google</span>
+        </div>
+      </div>
+    </section>
+  </main>
+
+  <footer class="wrapper flex">
+    <a href="index.html">
+      <img class="logo" src="assets/images/logo2.png" alt="logo">
+      <p class="max-font-26">SP Fitness<br>center 2023</p>
+    </a>
+    <p class="max-font-26">
+      2023 | SuperPower Fitness center
+      <span>
+        <br><br>
+        Lorem ipsum dolor sit amet consectetur adipisicing elit. Vitae minima recusandae voluptate. Numquam esse cumque consequatur culpa, veniam sunt, vero repellat, consequuntur itaque maiores fugit sequi ullam exercitationem repudiandae magni?
+      </span>
+    </p>
+  </footer>
```

Comment:
- [ ] Great job for adding the footer but in the Mobile version in Home page if you check the given design there is no partner section and footer so I recommend to remove them from the home page in the mobile version 💯🔥🚀✨

- Your design

![image](https://user-images.githubusercontent.com/58553711/235008729-91b2643b-e499-4a60-89f8-a78bdb9e082a.png)

- Given
![image](https://user-images.githubusercontent.com/58553711/235008775-77c59901-95d4-4e8c-b4c3-c59b7c9bd200.png)